rpcs3/rpcs3qt/game_list_frame.cpp

```diff
@@ -1010,7 +1010,12 @@ void game_list_frame::ShowContextMenu(const QPoint &pos)
 	// Make Actions
 	QMenu menu;
 
-	const bool is_current_running_game = (Emu.IsRunning() || Emu.IsPaused()) && current_game.serial == Emu.GetTitleID();
+	static const evaluate_is_current_game_running = [](const std:L:string& serial)
```

Comment:
There is no reason for the "evaluate_" prefix.
This only makes it harder to read